Src/VimWpf/Implementation/CommandMargin/CommandMarginController.cs

```diff
@@ -74,6 +74,7 @@ internal void Clear()
         private readonly IEditorFormatMap _editorFormatMap;
         private readonly IClassificationFormatMap _classificationFormatMap;
         private readonly ICommonOperations _commonOperations;
+        private readonly IClipboardDevice _clipboardDevice;

```

Comment:
In order to make this change testable, we needed to supply the command margin controller with a clipboard service. When used as an extension, the clipboard device is the real clipboard; When running unit tests, the clipboard device is a testable mock of the real clipboard.